prow/scripts/kyma-testing.sh

```diff
@@ -119,7 +87,7 @@ function main() {
   export INSTALL_DIR=${TMP_DIR}
   if ! [[ -x "$(command -v kyma)" ]];
```

Comment:
maybe we should add this check directly to the `kyma::install_cli`? Right now this function would install Kyma-cli regardless if it's available or not to the `/tmp/bin` directory. I'm not sure if we want to have two binaries in the PATH.

Reply:
We could move this check to `kyma::install_cli` and warn there is a version already installed and it may be out of date

Reply:
maybe not a warning but an info that it's installed and show its version

Reply:
Check moved to `kyma::install_cli`